moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -128,11 +129,19 @@ const ProjectActivityLog = () => {
     return data?.moped_activity_log?.length ?? 0;
   };
 
+  /**
+   * Returns True if the field should be a generic type (i.e., maps, objects)
+   * @param {string} field - The field name (column name)
+   * @return {boolean} - True if the field is contained in the ProjectActivityLogGenericDescriptions object
+   */
+  const isFieldGeneric = field =>
+    field in ProjectActivityLogGenericDescriptions;
```

Comment:
I need to remember to use the `in` keyword more. 👍

Reply:
I found out it works out of luck, that's the syntax in python, it totally works in JavaScript too! :)